test/Feature/HLSLLib/min.32.test

```diff
@@ -0,0 +1,177 @@
+#--- source.hlsl
+StructuredBuffer<float4> X0 : register(t0);
+StructuredBuffer<float4> Y0 : register(t1);
+StructuredBuffer<int4> X1 : register(t2);
+StructuredBuffer<int4> Y1 : register(t3);
+StructuredBuffer<uint4> X2 : register(t4);
+StructuredBuffer<uint4> Y2 : register(t5);
+
+RWStructuredBuffer<float4> Out0 : register(u6);
+RWStructuredBuffer<int4> Out1 : register(u7);
+RWStructuredBuffer<uint4> Out2 : register(u8);
+
+
+[numthreads(1,1,1)]
+void main() {
+  // float
+  Out0[0] = min(X0[0], Y0[0]);
+  Out0[1] = float4(min(X0[1].xyz, Y0[1].xyz), min(X0[1].w, Y0[1].w));
+  Out0[2] = float4(min(X0[2].xy, Y0[2].xy), min(X0[2].zw, Y0[2].zw));
+  Out0[3] = min(X0[3], Y0[3]);
+  Out0[4] = min(half4(1.0, -1.0, 31408, -415), half4(-1.0, 1.0, 1.5, 129.5));
+  
+  // int
+  Out1[0] = min(X1[0], Y1[0]);
+  Out1[1] = int4(min(X1[1].xyz, Y1[1].xyz), min(X1[1].w, Y1[1].w));
+  Out1[2] = int4(min(X1[2].xy, Y1[2].xy), min(X1[2].zw, Y1[2].zw));
+  Out1[3] = min(X1[3], Y1[3]);
+  Out1[3] = min(int4(-2147483648, -10, 10, 2147483647), int4(0, 10, 10, 0));;
+  
+  // uint
+  Out2[0] = min(X2[0], Y2[0]);
+  Out2[1] = uint4(min(X2[1].xyz, Y2[1].xyz), min(X2[1].w, Y2[1].w));
+  Out2[2] = uint4(min(X2[2].xy, Y2[2].xy), min(X2[2].zw, Y2[2].zw));
+  Out2[3] = min(X2[3], Y2[3]);
+  Out2[3] = min(uint4(0, 0, 10, 10000), uint4(0, 256, 4, 10001));
+}
+//--- pipeline.yaml
+
+---
+Shaders:
+  - Stage: Compute
+    Entry: main
+    DispatchSize: [1, 1, 1]
+Buffers:
+  - Name: X0
+    Format: Float32
+    Stride: 16
+    Data: [ -inf, -inf, -inf, -inf, inf, inf, inf, inf, NaN, NaN, NaN, NaN, 1.0, 1.0, -1.0, -1.0 ]

```

Comment:
i think we decided we want to avoid inf/nan like values and just use "regular values"